features/admin.roles.v2/components/application-roles.tsx

```diff
@@ -429,20 +429,26 @@ export const ApplicationRoles: FunctionComponent<ApplicationRolesSettingsInterfa
                                         value: BasicRoleInterface[],
                                         getTagProps: AutocompleteRenderGetTagProps
                                     ) => value.map((option: BasicRoleInterface, index: number) => (
-                                        <Chip
-                                            { ...getTagProps({ index }) }
-                                            key={ index }
-                                            label={ option.name }
-                                            activeOption={ activeOption }
-                                            setActiveOption={ setActiveOption }
-                                            variant={
-                                                initialSelectedRoles?.find(
-                                                    (role: BasicRoleInterface) => role.id === option.id
-                                                )
-                                                    ? "solid"
-                                                    : "outlined"
-                                            }
-                                        />
+                                        <>
```

Comment:
Tracked in an issue to validate the usages and remove later: https://github.com/wso2/product-is/issues/21351